frontend/javascripts/test/backend-snapshot-tests/annotations.e2e.ts

```diff
@@ -1,3 +1,4 @@
+import "test/mocks/lz4";
```

Comment:
Instead of importing the mock in every test file, I think it should work and would be more foolproof to import it once in `frontend/javascripts/test/enzyme/e2e-setup.ts` or do you think the import order could be problematic, then? :thinking: 

Sorry to comment on the mocking again. It's no big deal if you leave it as-is.